pkg/command/service/common.go

```diff
@@ -67,11 +69,50 @@ type measureResult struct {
 	queueProxyStartedSum              float64
 	userContrainerStartedSum          float64
 	deploymentCreatedSum              float64
+	Result                            result
```

Comment:
I would arrange the private and public fields together. Or better have different structs and just create a top level that includes the others. Eg.,

```go
type Sums struct {
... ReadySum float64
...
}
...

type measureResult struct {
 Sums
...
}
```

Reply:
Thanks, that's a good point. I'll open another PR to arrange these fields.

Reply:
@maximilien Hi, I've submit [a code change](https://github.com/knative-sandbox/kperf/pull/58), is it the correct direction?